iac/terraform/aws/cognito/cognito.tf

```diff
@@ -56,6 +56,11 @@ resource "aws_cognito_user_pool" "user_pool" {
 
 }
 
+resource "aws_cognito_user_pool_domain" "name" {
+  domain = "bloodconnect"
```

Comment:
why is the domain hardcoded?

Reply:
It's not the full domain name and It's used just one place. That's why I didn't declared another variable for this.

Reply:
changed the code. Used variable
https://github.com/CraftsmenLtd/BloodConnect/pull/163/files#diff-de0ee50e8abad6227b92d1269d6b63da2edc830def52ba433c7b727cc65c9bacR60-R63